system/jlib/jevent.hpp

```diff
@@ -34,10 +35,15 @@ enum EventType : byte
     EventIndexLookup,
     EventIndexLoad,
     EventIndexEviction,
+    EventDaliChangeMode,
+    EventDaliCommit,
     EventDaliConnect,
-    EventDaliRead,
-    EventDaliWrite,
-    EventDaliDisconnect,
+    EventDaliEnsureLocal,
+    EventDaliGet,
+    EventDaliGetChildren,
+    EventDaliGetChildrenFor,
+    EventDaliGetElements,
+    EventDaliSubscribe,
```

Comment:
I know it's a work in progress, but this shows how easy it is to invalidate all existing data files by disregarding the admonition to not change order or remove items. It will also conflict with HPCC-33729. I suggest not merging that PR until after it can be updated for this change.

Reply:
Agreed - but at the moment there are several other pending changes that will invalidate the format (e.g. compression).